controllers/installation_controller.go

```diff
@@ -404,3 +411,27 @@ func (r *InstallationReconciler) retry(ctx context.Context, log logr.Logger, ins
 	log.V(Log4Debug).Info("Retried associated porter agent action", "name", "retry", action.Name, retry)
 	return nil
 }
+
+func (r *InstallationReconciler) applyDeletionPolicy(ctx context.Context, log logr.Logger, inst *v1.Installation, policy string) error {
+	log.V(Log5Trace).Info("updating deletion policy")
+	annotations := inst.GetAnnotations()
+	if len(annotations) < 1 {
+		annotations = map[string]string{}
+	}
+
+	if _, ok := annotations[v1.PorterDeletePolicyAnnotation]; !ok {
+		annotations[v1.PorterDeletePolicyAnnotation] = v1.PorterDeletePolicyDelete
+		inst.SetAnnotations(annotations)
+		return r.Update(ctx, inst)
+	}
+
+	if policy != v1.PorterDeletePolicyOrphan && policy != v1.PorterDeletePolicyDelete {
+		annotations[v1.PorterDeletePolicyAnnotation] = v1.PorterDeletePolicyDelete
```

Comment:
Can you put a trace log in for this case so we can see if we're ever trying to set the delete policy to something we don't expect?

Reply:
Definitely,  I passed in the logger but I never used it 🤕

Reply:
The force push doesn't show that this is updated but it is.  Using a debug log to ensure that the policy they entered is not that of the two policies we have.  Resolving this.